shadowsocks/udp.go

```diff
@@ -214,6 +215,12 @@ func (m *natmap) Get(key string) (net.PacketConn, string) {
 	return entry.conn, entry.clientLocation
 }
 
+// Refresh the NAT mapping.  This should be called on every write for
+// outbound-refresh behavior.
```

Comment:
Please add a reference to https://tools.ietf.org/html/rfc4787#section-4.3

While at it, could you also add a comment to the 5 min flag saying it's the RFC recommendation?

Reply:
Done.  I also moved the default NAT timeout into a named constant.